src/Umbraco.Web.UI.Client/src/common/resources/emailmarketing.resource.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @ngdoc service
+ * @name umbraco.resources.emailMarketingResource
+ * @description Used to add a backoffice user to Umbraco's email marketing system, if user opts in
+ *
+ *
+ **/
+function emailMarketingResource($http, umbRequestHelper) {
+
+    // LOCAL
+    // http://localhost:7071/api/EmailProxy
+
+    // DEV
+    // https://devwecmsfunctions.azurewebsites.net/api/EmailProxy
+
+    // LIVE
+    // http://prwecmsfunctions.azurewebsites.net/api/EmailProxy
+
+    const emailApiUrl = 'https://devwecmsfunctions.azurewebsites.net/api/EmailProxy';
```

Comment:
We can't hardcode these urls to the azurewebsites url directly. We need a DNS layer in-between, so that we can swap the underlying implementation and not break the old already deployed emailmarketing.resource.js

Reply:
Feel free to give me a URL and set one up for me pretty please @pijemcolu ❤️